docs/installation/configuration.rst

```diff
@@ -209,7 +209,7 @@ Configure Worker named tokens to access external APIs:
 
 .. code-block:: bash
 
-  -e WORKER_HTTP_OAUTH2_TOKENS=token_name1=scope1,scope2,scope3:token_name2=scope1,scope2
+  -e WORKER_PLUGIN_HTTP_OAUTH2_TOKENS=token_name1=scope1,scope2,scope3:token_name2=scope1,scope2
```

Comment:
the other plugins still use the uid= so I believe this could be a global option.